### DISCUSSION TITLE
Multi cluster service mirroring UDP traffic

### DISCUSSION THREAD

**Discussion by andreavivaldi** [linkerd/linkerd2#9759] [Q&A]
Hi all, I'm new here and I'm new to LinkerD. I have a question: is it possible to use the Multi Cluster Service mirroring in the case of service and application that deals with only UDP traffic and not TCP?

Thanks in advance.
BR
Andrea

**Comment by kleimkuhler** [comment]
Linkerd does not currently support UDP. This isn't something we've recently discussed, but you can look at some previous discussion at #4723. Sorry that I cannot provide a more precise answer in terms of timeline, but the issue will at least stay open and we're aware of it.
